### PR TITLE
chore(www): Refactor `homepage-logo-banner`

### PR DESCRIPTION
**Pull request by fk**
Switch from template literal/string styles to [`sx`](https://theme-ui.com/sx-prop) and object styles.

### CODE REVIEW

**Comment by fk** [comment]
OK this is GTG 😅

**Comment by fk** [comment]
Hey YouTuuuuube link missing
https://www.youtube.com/watch?v=yVEgxDad0_w

**Comment by fk** [comment]
Yeah, thanks for calling it — I was kinda telling myself this would make the review easier, but … lemme fix this one now.
A bit short on time, but happy to do so with the two other PRs later today, too.

File: www/src/components/homepage/homepage-logo-banner.js [hunk 1]
```diff
@@ -2,66 +2,52 @@
...
-  padding-left: ${p => p.theme.space[3]};
-  padding-bottom: ${p => p.theme.space[4]};
-  ${mediaQueries.xxl} {
-    padding-bottom: ${p => p.theme.space[6]};
-  }
-  &::-webkit-scrollbar {
-    display: none;
-  }
-`
+const LogoGroup = props => (
```
Comment (fk) [inline www/src/components/homepage/homepage-logo-banner.js:32]: Btw. I feel like up to this object length I'd go inline without even thinking about it.
So … what are your thoughts re:"responsive arrays" @tesseralis?

Comment (tesseralis) [reply www/src/components/homepage/homepage-logo-banner.js]: Could you leave more details on this? I'm not sure what "responsive arrays" are

File: www/src/components/homepage/homepage-logo-banner.js [hunk 2]
```diff
@@ -86,20 +28,50 @@ const HomepageLogoBanner = () => {
   `)
 
   return (
-    <Section>
-      <Title>
+    <section
+      sx={{
+        borderBottom: t => `1px solid ${t.colors.ui.border}`,
```
Comment (tesseralis) [inline www/src/components/homepage/homepage-logo-banner.js:33]: I think you can do:

```suggestion
        borderBottom: t => `1px solid ${t.colors.ui.border}`,
        borderBottom: 1,
        borderColor: `ui.border`,
```

File: www/src/components/homepage/homepage-logo-banner.js (hunk 1 above)
Comment (fk) [reply www/src/components/homepage/homepage-logo-banner.js]: `[null, null, null, "20px"]` — https://theme-ui.com/sx-prop/#responsive-values

Comment (tesseralis) [reply www/src/components/homepage/homepage-logo-banner.js]: Ohhhhhhhh. Yeah they're great :P

File: www/src/components/homepage/homepage-logo-banner.js (hunk 2 above)
Comment (fk) [reply www/src/components/homepage/homepage-logo-banner.js:33]: ha just pushed right after you merged — needed a `borderStyle: "solid"` to make that work

File: www/src/components/homepage/homepage-logo-banner.js (hunk 1 above)
Comment (fk) [reply www/src/components/homepage/homepage-logo-banner.js]: For me there are cases where I prefer to not use them, … and also the more `null`s, the more torn I am :)

Comment (fk) [reply www/src/components/homepage/homepage-logo-banner.js]: Btw. did you see the v1 roadmap?
https://github.com/system-ui/theme-ui/issues/832

**Pull request merged**